pallets/collator-assignment/src/mock.rs

```diff
@@ -162,10 +168,26 @@ impl tp_traits::GetSessionContainerChains<u32> for ContainerChainsGetter {
     }
 }
 
+pub struct RotateCollatorsEveryNSessions<Period>(PhantomData<Period>);
```

Comment:
Isnt this what we also use in the runtime? if so, can we make it a type that we use both for tests and runtime-?

Reply:
Ok I can move this to primitives